src/modules/app/components/routes.jsx

```diff
@@ -57,15 +57,12 @@ export default class Routes extends Component {
         });
         break;
       case ACCOUNT:
-        viewProps = {
-          loginMessageLink: p.links.loginMessageLink,
-          account: p.loginAccount,
-          onChangePass: p.loginAccount.onChangePass,
-          authLink: (p.links && p.links.authLink) || null,
-          onAirbitzManageAccount: p.loginAccount.onAirbitzManageAccount
-        };
-        import('modules/account/components/account-view').then((module) => {
+        import('modules/account/container').then((module) => {
           const AccountView = module.default;
+          viewProps = {
+            loginMessageLink: p.links.loginMessageLink,
```

Comment:
This might not have been addressed yet due to being in-process, but we may want to move all values passes via props to the container as well.